src/constants.ts

```diff
@@ -0,0 +1,22 @@
+import { IGridLayout } from './state/interfaces/controls';
+
+/**
+ * Offers constant information values to use in an application.
+ */
+export module InteractiveConstants {
```

Comment:
Alright, feel free to change it then!
I won't be able to for the next two weeks.